### ISSUE TITLE
[BUG][e2e] base64 tool syntax not working on Windows and some MacOS

### ISSUE THREAD

**Issue by jsliacan** [crc-org/crc#3823]
### General information

  * macOS and Windows

### CRC version
```bash
CRC version: 2.26.0+233df0
OpenShift version: 4.13.9
Podman version: 4.4.4
```

The `base64` tools that we have on Win11Pro and MacOS 13.0.1 do not like the standalone `-` in 
```
base64 -d - > ca.crt
```
found in [story_openshift](https://github.com/crc-org/crc/blob/233df0ee1c05f91d49dbe72dc58a2288ac09f30f/test/e2e/features/story_openshift.feature#L7C10-L7C10).

The complaint: 
```
23:39:22.163347    bash: oc config view --raw -o jsonpath="{.clusters[?(@.name=='api-crc-testing:6443')].cluster.certificate-authority-data}"| base64 -d - > ca.crt
23:39:22.169552  stderr: base64: invalid argument -
```

**Comment by adrianriobo** [comment]
Introduced by #3741

**Comment by adrianriobo** [comment]
Closed by #3841
